core/src/test/java/org/apache/iceberg/rest/RESTCatalogAdapter.java

```diff
@@ -347,8 +347,12 @@ public <T extends RESTResponse> T handleRequest(
 
       case LOAD_TABLE:
         {
+          RESTSessionCatalog.SnapshotMode snapshotMode =
+              RESTSessionCatalog.SnapshotMode.valueOf(
+                  PropertyUtil.propertyAsString(
+                      vars, "snapshots", RESTSessionCatalog.SnapshotMode.ALL.name()));
```

Comment:
shouldn't this be called `snapshot-loading-mode` rather than `snapshots`?

Reply:
`snapshot-loading-mode` is the property name in `RESTSessionCatalog` to init snapshotMode.  `snapshots` is the name of queryParams when REST client sending requests.
```java
 enum SnapshotMode {
    ALL,
    REFS;

    // REST clients use it as queryParams
    Map<String, String> params() {
      return ImmutableMap.of("snapshots", this.name().toLowerCase(Locale.US));
    }
  }
```